pkgs/tools/misc/xdg-ninja/default.nix

```
@@ -0,0 +1,38 @@
+{ lib, stdenvNoCC, fetchFromGitHub, makeWrapper, jq, glow }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "xdg-ninja";
+  version = "0.2.0.1";
+
+  src = fetchFromGitHub {
+    owner = "b3nj5m1n";
+    repo = "xdg-ninja";
+    rev = "v${version}";
+    sha256 = "sha256-ZyqxMlyCB8gEsZTVrxgLdW/mQ/4xeTHTK+lDKIzYs6I=";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    runHook preInstall
+    install -Dm755 xdg-ninja.sh "$out/share/xdg-ninja/xdg-ninja.sh"
+    install -Dm644 programs/* -t "$out/share/xdg-ninja/programs"
+
+    mkdir -p "$out/bin"
+    ln -s "$out/share/xdg-ninja/xdg-ninja.sh" "$out/bin/xdg-ninja"
+
+    wrapProgram "$out/bin/xdg-ninja" \
+      --prefix PATH : "${lib.makeBinPath [ glow jq ]}"
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description =
+      "A shell script which checks your $HOME for unwanted files and directories.";
```

```suggestion
    description = "A shell script which checks your $HOME for unwanted files and directories.";
```

---

https://nixos.org/manual/nixpkgs/unstable/#var-meta-description
> Don’t include a period at the end.

```suggestion
    description = "A shell script which checks your $HOME for unwanted files and directories";
```